bundles/binding/org.openhab.binding.smlreader/src/main/java/org/openhab/binding/smlreader/internal/SmlReaderGenericBindingProvider.java

```diff
@@ -0,0 +1,183 @@
+/**
+ * Copyright (c) 2010-2013, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.smlreader.internal;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.openhab.binding.smlreader.SmlReaderBindingProvider;
+import org.openhab.core.binding.BindingConfig;
+import org.openhab.core.items.Item;
+import org.openhab.core.library.items.NumberItem;
+import org.openhab.core.library.items.StringItem;
+import org.openhab.model.item.binding.AbstractGenericBindingProvider;
+import org.openhab.model.item.binding.BindingConfigParseException;
+
+/**
+ * This class is responsible for parsing the binding configuration.
+ * 
+ * <p>Here are some examples for valid binding configuration strings: 
+ * <ul> 
+ * <li><code>{ smlreader="edl300#1-0:2.8.0#JS:myConversion.js" }</code></li> 
```

Comment:
example doesn't seem to match the given REGEX (it should use '()' instead right?)